css/css-quiz.md

```diff
@@ -2088,3 +2090,36 @@ img {
   margin-top: 150px;
 }
 ```
+  
+#### Q133. How would the ```<div>``` with the class .child be positioned within its container?
+```html
+<style>
+.container{
+height: 200px;
+width: 200px;
+}
+.child{
+margin: auto 0;
+height: 100px;
+}
+</style>
+<div class="container">
+    <div class="child">...</div>
+</div>
+```
+
+- [ ] horizontally centered
+- [ ] vertically centered, spanning the container's width
+- [ ] at the bottom of the container, spanning the container's width
+- [x] at the top of the container, spanning the container's width
+
+[Reference](https://codepen.io/harsh1x4/pen/poVaQVz)
+
+#### Q134. By default, a CSS transition is __
+
+- [ ] asymmetric, so that the transition going from the initial state to the end state is different from the transition going from the end state back to the initial state.
```

Comment:
The answer for Q134. will be (C)
u can refer [!reference](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions)
Please update it and PR again.

Reply:
Thank You.✌